draft-ietf-rats-architecture.md

```diff
@@ -679,13 +679,7 @@ In the background-check model, this Evidence may also be revealed to Relying Par
 
 ## Evidence
 
-Today, Evidence tends to be highly device-specific, since the information in the Evidence
-often includes vendor-specific information that is necessary to fully describe the manufacturer
-and model of the device including its security properties, the health
-of the device, and the level of confidence in the correctness of the information.
-Evidence is typically signed by the device (whether by hardware, firmware, or software on the
-device), and its appraisal in isolation would require Appraisal Policy to be based on
-device-specific details (e.g., a device public key).
+Evidence is a set of claims about the target device that reveals operational status, health, configuration or construction that have security relevance. Evidence is evaluated by a Verifier to establish its relevance, correctness and timeliness. Claims need to be collected in a manner that is reliable, that way if the target somehow can lie about them, the attesting environement is able to detect or circumvent it (lying). Evidence needs to be securely associated with the target device so that the Verifier cannot be tricked into accepting claims originating from a different target (that may be more trustworthy than the intended target). Evidence also must be protected from man-in-the-middle attackers who may observe, change or misdirect Evidence as it travels from Attester to Verifier. The timeliness of Evidence can be captured using claims that pinpoint the time or interval where changes in operational status, health and so forth occur.
```

Comment:
Please split into multiple source lines to make diffs and commenting in context easier

E.g. grammar: "reveals" -> "reveal"
Add oxford comma after "correctness"
Make "that way" start a new sentence
Typo "environement"
Avoid using the word "target"